baseline/client/n-back/frag/cue_2_wrong.html

```diff
@@ -0,0 +1,2 @@
+Hmm, that didn't look right. Please press the number "2" to start the zero-back task. <br>
```

Comment:
This should be "...to start the two-back task" rather than zero-back task.

Reply:
Ah, nice catch!